index.html

```diff
@@ -303,30 +311,24 @@
       <h2>Introduction</h2>
 
       <p>
-[=Credentials=] are integral to our daily lives; driver's licenses confirm
+[=Credentials=] are integral to our daily lives: driver's licenses confirm
 our capability to operate motor vehicles, university degrees assert our level
```

Comment:
Must be applied with subsequent change request
```suggestion
[=Credentials=] are integral to our daily lives: driver's licenses confirm
our capability to operate motor vehicles; university degrees assert our level
```